javascript/ql/lib/semmle/javascript/XML.qll

```diff
@@ -165,6 +174,9 @@ class XMLDTD extends XMLLocatable, @xmldtd {
   }
 }
 
+/** DEPRECATED: Alias for XmlDTD */
+deprecated class XMLDTD = XmlDTD;
```

Comment:
while we're at it, can we name this one `XmlDtd`?

Reply:
I going to rename more things in a later PR.
I'll include DTD in that one.